jerry-core/ecma/builtin-objects/ecma-builtin-json.c

```diff
@@ -1491,7 +1491,7 @@ ecma_builtin_json_object (ecma_object_t *obj_p, /**< the object*/
       member_str_p = tmp_str_p;
 
       /* 8.b.iii */
-      bool is_gap_empty = (ecma_string_get_length (context_p->gap_str_p) == 0);
+      bool is_gap_empty = (ecma_string_is_empty (context_p->gap_str_p));
```

Comment:
These could be placed inside their respective `if`s without an intermediate variable.